docs/rst/manual/getting_started/configuration/poly_new/guide.rst

```diff
@@ -0,0 +1,459 @@
+simulations. We can also specify that we want to run more simultaneous
+simulations, so it will run faster. This is configured in the queue system by
+specifying a :ref:`queue option <queue_option>` ``MAX_RUNNING`` for the
+``LOCAL`` queue, like this: ``QUEUE_OPTION LOCAL MAX_RUNNING 50``.
+
+After adding these two lines and changing the number of realizations, the
+config should look like this:
+
+    .. include:: with_parameters/poly.ert
+        :code:
+
+Running with sampled parameters
+*******************************
+Now you should delete the ``storage`` and ``poly_out`` folders from last run,
+so we know we are getting only new data.
+
+Launch ERT again. Notice that the config summary now specifies the name of the
+parameter set we defined. Then select Ensemble Experiment in the simulation
+mode, and start the simulation.
+
+When the simulations are done, you can now press the "Create Plot" button in
+the progress window or in the main window, and the Plotting window should open.
+Here you can now see the distributions of the three different parameters we
+created. They are named ``COEFFS:COEFF_A``, ``COEFFS:COEFF_B`` and
+``COEFFS:COEFF_C``, with the parameter set name first, then a colon and then
+the name of the specific parameter.
+
+You should see something similar to this:
+
+    .. image:: with_parameters/plots.png
+
+Play around and look at the different plots.
+
+Inspecting the parameters and results
+*************************************
+
+Inside each of the runpaths you should now be able to find the instantiated
+parameter template files ``coeffs.json``. Looking at them (e.g. with
+``cat poly_out/real_4/iter_0/coeffs.json``) you should see something like
+this::
+
+    {
+        "a": 0.830303,
+        "b": 1.69181,
+        "c": 0.114524
+    }
+
+If you now look at the generated file ``poly_0.out`` in the runpaths you should
+also see that each simulation has yielded different results. Here is one
+possible output from running ``cat poly_out/real_0/iter_0/poly_0.out``::
+
+    2.23622
+    4.288035
+    6.83408
+    9.874355
+    13.40886
+    17.437595
+    21.96056
+    26.977755
+    32.48918
+    38.494835
+
+In the next section, we will see how to describe the results to ERT, and how to
+specify some observations that we wish ERT to optimise towards.
+
+Reading simulation results
+--------------------------
+We have to tell ERT where to find the results of our simulations. For general
+data like we have in this example, we use the :ref:`GEN_DATA <gen_data>`
+keyword. Add this line to the ``poly.ert`` file::
+
+    GEN_DATA POLY_RES RESULT_FILE:poly_%d.out REPORT_STEPS:0 INPUT_FORMAT:ASCII
+
+The arguments of ``GEN_DATA``:
+
+POLY_RES
+    Name of this result set.
+RESULT_FILE:poly_%d.out
+    File with results of simulation. The ``%d`` is always ``0``, but needs to
+    be specified. (it was used in earlier ERT versions)
+REPORT_STEPS:0
+    TODO
```

Comment:
Should there be a TODO here? If so the formatting should be different?

Reply:
No, please inform we what makes sense to write there

Reply:
From the manual: 
```The REPORT_STEPS:0 is tightly bound to the %d integer format specifier in the result file - at load time the %d is replaced with the integer values given in the REPORT_STEPS: option, for the example given above that means that %d will be replaced with 0 and ERT will look for the file SimulatedWOC_0.txt. In principle it is possible to configure several report steps like: REPORT_STEPS:0,10,20 - then ERT will look for all three files SimulatedWOC_0.txt, SimultedWOC_10.txt and SimulatedWOC_20.txt. It is quite challenging to get this right, and the recommendation is to just stick with one result file at report step 0 3, in the future the possibility to load one keyword GEN_DATA for multiple report steps will probably be removed, but for now the GEN_DATA configuration is quite strict - it will fail if the RESULT_FILE attribute does not contain a %d.```
Think I would write something like: Tied to time map, for most cases this will be 0.